src/directives/formly-form.test.js

```diff
@@ -933,12 +933,19 @@ describe('formly-form', () => {
 
       const listener = sinon.spy()
       const expression = sinon.spy()
+      const deepLinkField = getNewField()
+      deepLinkField.key = 'foo.bar'
+      deepLinkField.watcher = {
+        listener,
+        expression: sinon.spy(),
```

Comment:
Not quite. We don't need to compile and digest more than once. But the other stuff is probably good.